v1/examples/schema_example.yaml

```diff
@@ -0,0 +1,129 @@
+# LPG YAML Schema Options - examples
+
+# Version 1 - simple
```

Comment:
Simple version is for interoperability (sharing info between different domains, e.g., business and research)

Reply:
USA-CASE: Share the understanding as quickly as possible